src/Transmogrification.cpp

```diff
@@ -354,7 +354,6 @@ TransmogTrinityStrings Transmogrification::Transmogrify(Player* player, uint64 i
                 {
                     if (!player->HasEnoughMoney(cost))
                         return LANG_ERR_TRANSMOG_NOT_ENOUGH_MONEY;
-                    player->ModifyMoney(-cost, false);
```

Comment:
Would be better to keep this and remove the other one.
https://github.com/TrinityCore/TrinityCore/commit/8fbc00f26a428ee05762ecb2bfc99e290a56d65c

Reply:
Can we still merge this for now?

Reply:
I would rather have it like this: https://github.com/azerothcore/mod-transmog/pull/8
But I dont have access to edit this PR nor merge this anyways.